### PR TITLE
mx - readme.md fix - fix broken image with reth repo official banner 

### PR DESCRIPTION
**Pull request by m0ham3dx**
# PR Description 

- Main index.html page page has a broken image link 
- This was fixed with [`reth official repo banner`](https://raw.githubusercontent.com/m0ham3dx/reth/main/assets/reth.jpg)

# Impact 

- No broken image in the cover page of the reth official book

### CODE REVIEW

**Review by mattsse — COMMENTED** [review]
does referencing `../assets/reth.jpg`work?

**Comment by m0ham3dx** [comment]
> does referencing `../assets/reth.jpg`work?

yes it [does](https://i.imgur.com/7OdvktV.png) .Please advice what I should do with this pr ?

File: book/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # reth Book
 
-<img src="images/" style="border-radius: 20px">
+<img src="https://raw.githubusercontent.com/m0ham3dx/reth/main/assets/reth.jpg" style="border-radius: 20px">
```
Comment (mattsse) [inline book/README.md:3]: can we change this to a relative import `../assets/reth.jpg`?

**Review by mattsse — APPROVED** [review]

File: book/README.md (hunk 1 above)
Comment (m0ham3dx) [reply book/README.md]: changes implemented -

Comment (m0ham3dx) [reply book/README.md]: it seem 
```
<img src="../assets/reth.jpg" style="border-radius: 20px">
``` 
doesnt render for some reason when pushed to github pages. When I was reading the book on my local repo 

But it works when the image is imported as a full url 
```
https://raw.githubusercontent.com/paradigmxyz/reth/main/assets/reth.jpg
```
When the book gets published eventually, and the image is broken, I will submit another pr 

if it doesnt, then i will resolve this conversation. Until then no more prs on this issue from me

@mattsse

Comment (mattsse) [reply book/README.md]: okay, let's do the full url then

mind submitting another PR?

Comment (m0ham3dx) [reply book/README.md]: PR #839

**Pull request merged**